js/app.js

```diff
@@ -1,17 +1,16 @@
-import React, { Component } from 'react';
+import React from 'react';
+import * as firebase from 'firebase';
 import AppNavigator from './appNavigator';
 
-class App extends Component {
+firebase.initializeApp({
+  apiKey: 'AIzaSyCY5F5QqKqaVTvwvQQf5EQgu2apNIfG1-0',
+  authDomain: 'justarrivedp2p.firebaseapp.com',
+  databaseURL: 'https://justarrivedp2p.firebaseio.com',
+  storageBucket: 'justarrivedp2p.appspot.com',
+  //messagingSenderId: '981762065581',
```

Comment:
For now Im not sure what the messagingSenderId is even used for. Have to look further into what it is.